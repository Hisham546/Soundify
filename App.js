import * as React from 'react';
import { NavigationContainer } from '@react-navigation/native';
import { createNativeStackNavigator } from '@react-navigation/native-stack';

import Routes from './src/routes';
const Stack = createNativeStackNavigator();

const App = () => {
   return (
      <Routes />

   );
};

export default App;