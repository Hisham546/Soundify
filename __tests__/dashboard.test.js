/**
 * @format
 */

import 'react-native';
import React from 'react';
import App from '../App';
import Dashboard from '../src/dashboard/dashboard';

// Note: import explicitly to use the types shiped with jest.
import { it } from '@jest/globals';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
   const tree = renderer.create(<Dashboard />).toJSON();
   expect(tree).toMatchSnapshot();
});
