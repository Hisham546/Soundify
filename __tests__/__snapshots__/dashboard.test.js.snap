// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "black",
      "flex": 1,
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "borderBottomColor": "gray",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "height": 106.5,
        "width": 750,
      }
    }
  >
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "opacity": 1,
        }
      }
    >
      <Text
        allowFontScaling={false}
        selectable={false}
        style={
          [
            {
              "color": "white",
              "fontSize": 46.5,
            },
            {
              "marginLeft": 22.5,
            },
            {
              "fontFamily": "Material Design Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            {},
          ]
        }
      >
        Û∞çú
      </Text>
    </View>
    <Text
      style={
        {
          "color": "white",
          "fontSize": 26.5,
          "fontWeight": "500",
          "marginLeft": 22.5,
        }
      }
    >
      Soundify
    </Text>
  </View>
  <View
    style={
      {
        "height": 1160.58,
        "width": 75000,
      }
    }
  />
</View>
`;
